### ISSUE TITLE
Incompatible with GDAP

### ISSUE THREAD

**Issue by JacobErnst98** [lwhitelock/HuduM365Automation#23]
M365-Hudu-Standalone fails with errors about invalid_grant after moving to GDAP.

**Comment by jn1102** [comment]
Yup. Same here. Let me know if you figure it out. I may have some time to play with it in a few weeks myself. I'll see if I can figure it out.

**Comment by mpowelltech** [comment]
I was having a similar issue but fixed it by granting admin consent to the application in the clients' tenants.
Used CIPP to do this with the Application Approval feature.

More info here: https://tminus365.com/my-automations-break-with-gdap-the-fix/
